client/packages/electron/updateElectronVersion.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+fs.readFile('../../../package.json', (err, rootPackageJson) => {
+  if (err) throw err;
+  const rootPackage = JSON.parse(rootPackageJson);
+
+  fs.readFile('./package.json', (err, packageJson) => {
+    if (err) throw err;
+
+    const package = JSON.parse(packageJson);
+    package.version = rootPackage.version;
+
+    fs.writeFile('./package.json', JSON.stringify(package, 2), err => {
```

Comment:
I think it's good to add a comment above version in package.json: "//": "Version is automatically updated when electron is built, see 'make' npm script"